src/components/ExportCSV/ExportCSVForm/ExportCSVFormMobile.jsx

```diff
@@ -550,16 +599,41 @@ const ExportCSVForm = ({ formID }) => {
             </DrawerBody>
           </DrawerContent>
         </Drawer>
-      </form>
 
-      <div className={styles['buttons-container']}>
-        <Button border="1px" borderColor="#CBD5E0" bg="white">
-          Cancel
-        </Button>
-        <Button textColor="white" bg="#345E80">
-          Preview CSV
-        </Button>
-      </div>
+        <Drawer onToggle={customCountryInputToggle} isOpen={customCountryInputOpen} size="full">
+          <DrawerContent>
+            <ChevronLeftIcon
+              className={styles['back-button']}
+              boxSize={7}
+              onClick={() => (isValid ? customCountryInputToggle() : console.log(isValid))}
```

Comment:
⚠️ **[eslint]** <[no-console](https://eslint.org/docs/rules/no-console)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected console statement.

Reply:
Could we make this an if statement instead of ternary and remove the console statement?